tests/test-backend-ops.cpp

```diff
@@ -3929,14 +3929,25 @@ static std::vector<std::unique_ptr<test_case>> make_test_cases_eval() {
         test_cases.emplace_back(new test_set(GGML_TYPE_I32, GGML_TYPE_I32, {6, 5, 4, 3}, dim));
     }
 
-    for (ggml_type type_src : {GGML_TYPE_F16, GGML_TYPE_F32}) {
+    // same-type copy
+    for (int nr = 1; nr < 4; ++nr) {
```

Comment:
What's the reason for this extra loop?

Reply:
It was meant to test multiple number of blocks in a row, but I had a bug. Should be fixed now.